libexec/rtld-elf/aarch64/rtld_machdep.h

```diff
@@ -1,6 +1,7 @@
 /*-
  * Copyright (c) 1999, 2000 John D. Polstra.
  * Copyright (c) 2014 the FreeBSD Foundation
+ * Copyright (c) 2020 Brett F. Gutstein
```

Comment:
If you're changing this, you probably want to add the copyright notes from the RISC-V rtld_machdep.h too and add CTSRD+ECATS since this is largely a copy.